src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h

```diff
@@ -13,7 +13,7 @@
  */
 
 #pragma once
-#include <unknwn.h>
+#include <unknown.h>
```

Comment:
this is breaking compilation on windows
can you remove this change ?

Reply:
Sorry about that. Will undo. It's extremely complex to view such changes across many files, tools such as `gitg` just cannot cope.

Reply:
The `<unknwn.h>` change was an error on my side.